framework/src/media/audio/audio_manager.c

```diff
@@ -1025,85 +1027,75 @@ uint32_t get_output_bytes_to_frame(unsigned int bytes)
 	return pcm_bytes_to_frames(g_audio_out_cards[g_actual_audio_out_card_id].pcm, bytes);
 }
 
-uint16_t get_max_audio_volume(void)
+audio_manager_result_t get_max_audio_volume(uint8_t *volume)
```

Comment:
Please add TODO here,
We will calculate max & current volume based on decibel value.
Now if device support only one level, maybe max & current will be 10 all the time.

Reply:
ToDo is added.